src/main/java/seedu/address/logic/commands/AddScoreCommand.java

```diff
@@ -15,7 +15,7 @@
 import seedu.address.model.score.Score;
 
 /**
- * Adds a person to the address book.
+ * Adds a score to a person identified using it's display index in the address book.
```

Comment:
Maybe instead of address book, can change it to student list?

`Adds a score to a student identified using it's display index in the student list`